Objects/regions.c

```diff
@@ -1315,6 +1329,48 @@ static PyObject *add_to_region(PyObject *obj, Py_region_ptr_t region)
     Py_RETURN_NONE;
 }
 
+int _Py_is_bridge_object(PyObject *op) {
+    Py_region_ptr_t region = Py_REGION(op);
+    if (IS_LOCAL_REGION(region) || IS_IMMUTABLE_REGION(region)) {
+        return false;
+    }
+
+    // It's not yet clear how immutability will interact with region objects.
+    // It's likely that the object will remain in the object topology but
+    // will use the properties of a bridge object. This therefore checks if
+    // the object is equal to the regions bridge object rather than checking
+    // that the type is `PyRegionObject`
+    return ((Py_region_ptr_t)((regionmetadata*)region)->bridge == (Py_region_ptr_t)op);
+}
+
+__attribute__((unused))
+static void regionmetadata_inc_lrc(regionmetadata* data) {
+    data->lrc += 1;
+}
+
+__attribute__((unused))
+static void regionmetadata_dec_lrc(regionmetadata* data) {
+    data->lrc -= 1;
+}
+
+static void regionmetadata_close(regionmetadata* data) {
+    data->is_open = 0;
+}
+
+__attribute__((unused))
+static void regionmetadata_unparent(regionmetadata* data) {
+    regionmetadata_set_parent(data, NULL);
+}
+
+__attribute__((unused))
+static int regionmetadata_is_root(regionmetadata* data) {
+    return regionmetadata_has_parent(data);
+}
+
+static regionmetadata* PyRegion_get_metadata(PyRegionObject* obj) {
+    return obj->metadata;
+}
```

Comment:
These methods have been removed in a previous commit PR, did you plan to reintroduce them?

If so, we should remove `PyRegion_get_metadata` as `Py_REGION_DATA` should be used to get the root of the merge tree instead

Reply:
Thanks, I'll remove them.

Reply:
I use _Py_is_bridge_object and regionmetadata_close. Can you see a problem with keeping those around?

Reply:
No, but then the `__attribute__((unused))` should be removed :)

Reply:
@xFrednet ☝️

Reply:
Woops, yeah ignore my comment ^^